evap/grades/templates/grades_course_view.html

```diff
@@ -12,66 +11,54 @@ <h3 class="mb-3">{{ course.name }} ({{ semester.name }})</h3>
         </div>
         <div class="card-body">
             {% if grade_documents %}
-                <table class="table table-striped table-vertically-aligned">
-                    <thead>
-                        <tr>
-                            <th style="width: 40%">{% trans 'Description' %}</th>
-                            <th style="width: 20%">{% trans 'Type' %}</th>
-                            <th style="width: 25%">{% trans 'Last edited' %}</th>
-                            <th style="width: 15%">{% trans 'Actions' %}</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {% for grade_document in grade_documents %}
-                            <tr id="grade_document-row-{{ grade_document.id }}">
-                                <td>{{ grade_document.description }}</td>
-                                <td>{{ grade_document.get_type_display }}</td>
-                                <td>{{ grade_document.last_modified_time }}, {% trans 'by' %} {{ grade_document.last_modified_user }}</td>
-                                <td>
-                                    <a href="{% url 'grades:download_grades' grade_document.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Download' %}"><span class="fas fa-download"></span></a>
-                                    {% if user.is_grade_publisher %}
-                                        <a href="{% url 'grades:edit_grades' grade_document.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Edit' %}"><span class="fas fa-pencil"></span></a>
-                                        <confirmation-modal confirm-button-class="btn-danger" data-delete-document="{{ grade_document.id }}">
-                                            <span slot="title">{% trans 'Delete grade document' %}</span>
-                                            <span slot="action-text">{% trans 'Delete grade document' %}</span>
-                                            <span slot="question">
-                                                {% blocktrans with description=grade_document.description %}
-                                                    Do you really want to delete the grade document <strong>{{ description }}</strong>?
-                                                {% endblocktrans %}
-                                            </span>
+                <form id="grade-document-deletion-form" custom-success method="POST" action="{% url 'grades:delete_grades' %}">
```

Comment:
I'd say the aspect that the only modal-confirmed button in this table is a "delete" button is somewhat accidental and can easily change in the future, so I wouldn't structure the HTML around it (-> I wouldn't put the whole table inside this form)

Can we immediately close the form and make the modal use `form="grade-document-deletion-form"` instead?